### DISCUSSION TITLE
Recommended Prisma Schema Type for Money

### DISCUSSION THREAD

**Discussion by danielyogel** [prisma/prisma#10160] [Q&A]
What is the recommended Prisma schema type for money/currency this days?
Should I just use 'Decimal'? 
What about Postgres native 'money' datatype?
Can I just use 'String'? (will be very comfortable - because components like 'react-currency-input-field' fire a 'string' value).

BTW - I'm using Prisma for months now, in a big project - and enjoys every second, thanks!

**Comment by austincrim** [comment]
Hi @danielyogel 👋, thanks for the question.

I think the Postgres `money` type would work great! You can use the Prisma schema's [native data type feature](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#postgresql-5) for this e.g.: 
```prisma
model Item {
  id    Int     @id @default(autoincrement())
  price Decimal @db.Money
}
```

Let me know if you run into any problems!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!

**Comment by ellharman** [comment]
For anyone coming to this now, Prisma's own docs suggest not to use the Postgres `money` type:

https://www.prisma.io/docs/postgres/query-optimization/recommendations/avoid-db-money#:~:text=The%20%40db.Money%20data%20type%20in%20PostgreSQL%20is%20not%20ideal%20for%20storing%20monetary%20values
